### PR TITLE
🇵🇹 Translations

### PR DESCRIPTION
**Pull request by SaraVieira**
Awesome stuff ! ❤️

This PR:

 - Adds PT translations
 - Fixes some grammar nazi stuff , sorry 😔
 - Adds translation for See the docs

Demo: https://dist-zsapvuqhaw.now.sh/

### CODE REVIEW

File: store/index.js [hunk 1]
```diff
@@ -4,15 +4,17 @@ import vuexI18n from 'vuex-i18n'
...
 // import translations of labels
 import translationsEn from '../src/locale/en/index'
+import translationsPTpt from '../src/locale/pt-pt/index'
 
-// create info about langauges
+// create info about languages
 const languages = (ctx => {
   let keys = ctx.keys()
   let values = keys.map(ctx)
-  // return keys.reduce((o, k, i) => { o[k] = values[i]; return o; }, {});
```
Comment (nicolo-ribaudo) [inline store/index.js]: Here at least the first `:` is needed

File: store/index.js [hunk 2]
```diff
@@ -4,15 +4,17 @@ import vuexI18n from 'vuex-i18n'
...
+  // return keys.reduce((o, k, i) => { o[k] = values[i] return o }, {})
```
Comment (nicolo-ribaudo) [inline store/index.js]: Here at least the first `;` is needed

Comment (SaraVieira) [reply store/index.js]: Added ! Thanks 👍

**Comment by sdras** [comment]
Thank you Sara! Nice to have you on here as a collaborator :)

**Pull request merged**